osbs/utils.py

```diff
@@ -340,9 +349,7 @@ def make_name_from_git(repo, branch, limit=53, separator='-', hash_size=5):
 
     repo = ''.join(repo_chars)
     branch = ''.join(branch_chars)
-    name = separator.join([repo, branch, hash_str])
-
-    return name
+    return separator.join(filter(None, (repo, branch, hash_str)))
```

Comment:
I think `filter()` is superfluous here, all members to be joined should be 'clean' by this point.